lib/src/main/java/org/team100/lib/commands/drivetrain/DriveWithProfile2.java

```diff
@@ -55,7 +56,7 @@ public DriveWithProfile2(
             SwerveDriveSubsystem drivetrain,
             HolonomicFieldRelativeController controller,
             SwerveKinodynamics limits) {
-        m_fieldRelativeGoal = fieldRelativeGoal;
+        m_fieldRelativeGoalSupplier = fieldRelativeGoal;
         m_swerve = drivetrain;
         m_controller = controller;
         m_limits = limits;
```

Comment:
these profile limits are unattainably fast, aren't they?  did you try making the profiles a lot slower, as i suggested?  "first make it perfect, then make it fast"?

Reply:
Fixed